From 1ad19ec769de6199609887e73db33c5e0f99ea25 Mon Sep 17 00:00:00 2001
From: John Crispin <john@phrozen.org>
Date: Wed, 20 Aug 2025 08:33:42 +0200
Subject: [PATCH 55/55] ugps: silence warnings

Adds quiet flag to UCI queries in ugps init script to suppress error messages when GPS configuration is absent.

Signed-off-by: John Crispin <john@phrozen.org>
---
 package/utils/ugps/files/ugps.init | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/package/utils/ugps/files/ugps.init b/package/utils/ugps/files/ugps.init
index e702b4ca44..f2ab2bc783 100644
--- a/package/utils/ugps/files/ugps.init
+++ b/package/utils/ugps/files/ugps.init
@@ -11,10 +11,10 @@ service_triggers() {
 }
 
 start_service() {
-	local tty="$(uci get gps.@gps[-1].tty)"
-	local baudrate="$(uci get gps.@gps[-1].baudrate || echo 0)"
-	local atime="$(uci get gps.@gps[-1].adjust_time)"
-	local disabled="$(uci get gps.@gps[-1].disabled || echo 0)"
+	local tty="$(uci -q get gps.@gps[-1].tty)"
+	local baudrate="$(uci -q get gps.@gps[-1].baudrate || echo 0)"
+	local atime="$(uci -q get gps.@gps[-1].adjust_time)"
+	local disabled="$(uci -q get gps.@gps[-1].disabled || echo 0)"
 
 	[ "$disabled" == "0" ] || return
 	[ "$tty" ] || return
-- 
2.34.1

